### PR TITLE
Dev/standardized ux

### PR DESCRIPTION
**Pull request by gaborkb**
Moved Cinemachine menu to GameObject.
Removed About page.

### CODE REVIEW

**Comment by JulienDidierjean** [comment]
Hi Krisztian, 
Thanks for putting me as a reviewer...
Here are my comments based on the screenshot you posted:
 - I would drop the "Create" from each line, none of the the other entries in this "Create" menu has one
 - I'm quite sure I would not put an enable/disable option there: "Storyboard Global Mute" 
I think we should put it under Cinemachine Preferences maybe... 
Question: Is this option not present in the UI of all the Cinemachine cameras ? Maybe it's enough...
Can you tell me why it's been added there in the first place ?
(I can post this comment in the cinemachine-core channel to trigger discussions if you want...

**Comment by gaborkb** [comment]
> * I would drop the "Create" from each line, none of the the other entries in this "Create" menu has one

Ok!

**Comment by gaborkb** [comment]
> * I'm quite sure I would not put an enable/disable option there: "Storyboard Global Mute"
>   I think we should put it under Cinemachine Preferences maybe...
>   Question: Is this option not present in the UI of all the Cinemachine cameras ? Maybe it's enough...
>   Can you tell me why it's been added there in the first place ?
>   (I can post this comment in the cinemachine-core channel to trigger discussions if you want...

Yes, please post it, as I am not sure what the answer is.

File: Editor/Editors/CinemachineStoryboardEditor.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ namespace Cinemachine.Editor
     [InitializeOnLoad]
     internal class CinemachineStoryboardMute
     {
-        const string StoryboardGlobalMuteMenuName = "Cinemachine/Storyboard Global Mute";
+        const string StoryboardGlobalMuteMenuName = "GameObject/Cinemachine/Storyboard Global Mute";
```
Comment (glabute) [inline Editor/Editors/CinemachineStoryboardEditor.cs:14]: Leave this in the Cinemachine menu for now (maybe as the only thing).
We need to create a preferences page to move it to.  We can save that for another PR.
(so just revert this change)

File: Extras~/ReleaseNotes.txt [hunk 2]
```diff
@@ -6,6 +6,7 @@
 - Bugfix (1256530) - disallow multiple components where appropriate
 - Bugfix: BlendList camera was incorrectly holding 0-length camera cuts
 - Bugfix (1174993) - CM Brain logo was not added to Hierarchy next to Main Camera after adding vcam for the first time after importing CM. 
+- UI update - Moved Cinemachine menu to GameObject Create menu and Right Click context menu for Hierachy window.
```
Comment (glabute) [inline Extras~/ReleaseNotes.txt:9]: This entire file can now be deleted, since it was only ever used in the About window.

**Review by glabute — CHANGES_REQUESTED** [review]
You also need to go through the manual (the Documentation~ folder) and correct any references to the Cinemachine menu

**Review by glabute — APPROVED** [review]

**Pull request merged**